packages/easy-email-editor/src/components/EditEmailPreview/index.tsx

```diff
@@ -9,6 +9,7 @@ import { DATA_ATTRIBUTE_DROP_CONTAINER, SYNC_SCROLL_ELEMENT_CLASS_NAME } from '@
 import { classnames } from '@/utils/classnames';
 import { ActiveTabKeys } from '@/components/Provider/BlocksProvider';
 import { useActiveTab } from '@/hooks/useActiveTab';
+import '@/assets/font/iconfont.css';
```

Comment:
I needed to move the font import here, I could not make it work from `@extension`